pkg/authorization/api/register.go

```diff
@@ -5,7 +5,7 @@ import (
 	"k8s.io/kubernetes/pkg/runtime"
 )
 
-const GroupName = ""
+const GroupName = "authorization.openshift.io"
```

Comment:
Not just yet.  The master can't handle it https://github.com/kubernetes/kubernetes/pull/27554 yet.